### ISSUE TITLE
Nuxt pages not being created when adding new collections in Directus

### ISSUE THREAD

**Issue by purplefoxdigital** [nuxtus/nuxtus#78]
After using create-nuxtus to use the Nuxtus boilerplate, when creating a collection in Directus, the corresponding Nuxt page is not being created.
Tested using both no authentication and static token authentication in Directus, with neither working
Nuxtus-cli also shows "No directus collections found" if I attempt to create the page manually.

**Comment by craigharman** [comment]
Hmmm. Any logs you can share (or a repo I can access and test)?
I'd expect an error in the console from `npm run dev`.

**Comment by purplefoxdigital** [comment]
I created a brand new app using `npx create-nuxtux test`
On startup after running `npm start`:

```
> test@0.0.1 start
> concurrently "npm run server" "npm run client"

[1] 
[1] > test@0.0.1 client
[1] > cd client && npm run dev -- -o
[1] 
[0] 
[0] > test@0.0.1 server
[0] > cd server && npx directus start
[0] 
[1] 
[1] > dev
[1] > nuxt dev -o
[1] 
[1] Nuxt 3.12.2 with Nitro 2.9.6
[1] 
[1]   ➜ Local:    http://localhost:3000/
[1]   ➜ Network:  use --host to expose
[1] 
[1] [nuxt:tailwindcss] ℹ Using Tailwind CSS from ~/assets/css/tailwind.css
[0] [14:03:28.049] WARN: "PUBLIC_URL" should be a full URL
[0] [14:03:28.051] WARN: Spatialite isn't installed. Geometry type support will be limited.
[0] [14:03:28.051] WARN: Upload directory (/home/jack/test/server/uploads) is not read/writeable!
[0] [14:03:28.093] INFO: Server started at http://0.0.0.0:8055
[1]   ➜ DevTools: press Shift + Alt + D in the browser (v1.3.6)
[1] 
[1] [nuxt:tailwindcss] ℹ Tailwind Viewer: http://localhost:3000/_tailwind/
[1] ℹ Vite server warmed up in 1342ms
[1] ℹ Vite client warmed up in 1896ms
[1] [nitro] ✔ Nuxt Nitro server built in 815 ms
```

Creating a new page in Directus:

```
[0] [14:06:23] POST /collections 200 94ms
[0] [14:06:23] POST /relations 200 97ms
[0] [14:06:23] POST /relations 200 95ms
[0] [14:06:23] GET /collections 200 20ms
[0] [14:06:23] GET /relations 200 29ms
[0] [14:06:23] GET /fields 200 57ms
[0] [14:06:23] GET /fields 304 54ms
[0] [14:06:23] GET /notifications?filter[_and][0][recipient][_eq]=6db7ea64-313b-487a-8299-4c0741940885&filter[_and][1][status][_eq]=inbox&aggregate[countDistinct]=id 304 54ms
[0] [14:06:23] GET /notifications?aggregate[countDistinct]=id 304 52ms
[0] [14:06:23] GET /collections/test 200 65ms
[0] [14:06:23] GET /notifications?limit=25&fields[]=id&fields[]=subject&fields[]=message&fields[]=collection&fields[]=item&fields[]=timestamp&sort[]=-timestamp&page=1&filter[_and][0][recipient][_eq]=6db7ea64-313b-487a-8299-4c0741940885&filter[_and][1][status][_eq]=inbox 304 51ms
[0] [14:06:23] GET /fields 304 100ms
[0] [14:06:24] PATCH /users/me/track/page 204 23ms
```

Added page to "Public" user in Directus permissions

```
[0] [14:08:00] POST /permissions/ 200 21ms
[0] [14:08:00] GET /permissions?filter[role][_null]=true 200 47ms
```

Attempting to access page on Nuxt site:

```
[1] 
[1]  WARN  [Vue Router warn]: No match found for location with path "/test"
[1] 
[1] 
[1]  WARN  [Vue Router warn]: No match found for location with path "/test"
[1] 
[1] 
[1]  WARN  [Vue Router warn]: No match found for location with path "/test"
[1] 
[1] 
[1]  WARN  [Vue Router warn]: No match found for location with path "/test"
[1] 
```

Running create in the Nuxtus CLI

```
                         _                                  _   _ 
  _ __    _   _  __  __ | |_   _   _   ___            ___  | | (_)
 | '_ \  | | | | \ \/ / | __| | | | | / __|  _____   / __| | | | |
 | | | | | |_| |  >  <  | |_  | |_| | \__ \ |_____| | (__  | | | |
 |_| |_|  \__,_| /_/\_\  \__|  \__,_| |___/          \___| |_| |_|
                                                                  
No Directus collections found.
```
```
[0] [14:11:08] POST /auth/login 200 508ms
[0] [14:11:08] POST /auth/login 200 507ms
[0] [14:11:08] GET /collections 200 16ms
```

What's interesting is it looks like the Nuxtus extention hasn't been installed in Directus, as the extentions page just shows "No Extentions"
![image](https://github.com/nuxtus/nuxtus/assets/56895948/8a1b9204-85ee-4193-99cf-ae604b95d7de)

**Comment by craigharman** [comment]
Thanks for the extra info. I think something has changed in the extension handling in recent DIrectus updates. I'm looking into it and will get back to you.

**Comment by craigharman** [comment]
Okay, so for some reason the directus hook isn't being installed to the correct location. I'll update the install scripts to fix this, but in the meantime you can do a couple of manual steps to get things working:

1. Move the folder from `/server/extensions/hooks/directus-extension-nuxtus-hook` to `/server/extensions/directus-extension-nuxtus-hook`
2. Move the index.js file into `dist/index.js`
3. In `/server/extensions/directus-extension-nuxtus-hook` place [this package.json](https://github.com/nuxtus/hook/blob/main/package.json)

Restart nuxtus and the hook will be installed.

**Comment by craigharman** [comment]
Fixed in [v2.1.2](https://github.com/nuxtus/create-nuxtus/releases/tag/v2.1.2) or follow the manual steps above to resolve on existing project.
